models/quality_measures/intermediate/cqm236_controlling_blood_pressure/quality_measures__int_cqm236__frailty.sql

```diff
@@ -0,0 +1,264 @@
+{{ config(
+     enabled = var('quality_measures_enabled',var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False))))
+   )
```

Comment:
Hey @deepsonshrestha, can you add an ` | as_bool` to the config logic for all of your new models? You can take a look at the final models to see how it works. We recently found a bug and had to add this fix.

Reply:
`| as bool` has been added to all the config models